demos/render/lv_demo_render.c

```diff
@@ -778,10 +778,10 @@ static void blend_mode_cb(lv_obj_t * parent)
     /*Make the parent darker for additive blending*/
     lv_obj_set_style_bg_color(parent, lv_color_hex(0x808080), 0);
 
-    static uint8_t buf_rgb565[LV_CANVAS_BUF_SIZE(36, 30, 16, LV_DRAW_BUF_STRIDE_ALIGN)];
-    static uint8_t buf_rgb888[LV_CANVAS_BUF_SIZE(36, 30, 24, LV_DRAW_BUF_STRIDE_ALIGN)];
-    static uint8_t buf_xrgb8888[LV_CANVAS_BUF_SIZE(36, 30, 32, LV_DRAW_BUF_STRIDE_ALIGN)];
-    static uint8_t buf_argb8888[LV_CANVAS_BUF_SIZE(36, 30, 32, LV_DRAW_BUF_STRIDE_ALIGN)];
+    static LV_ATTRIBUTE_MEM_ALIGN uint8_t buf_rgb565[LV_CANVAS_BUF_SIZE(36, 30, 16, LV_DRAW_BUF_STRIDE_ALIGN)];
```

Comment:
We should use draw buffer interface to make sure draw buffer meets GPU's requirement.

It's especially important now since we'll have Nema GPU and VG-Lite GPU that both have such requirements.

Based on the watch development experience, static buffer really is easily misused and took hours to debug(the days we have not enough param checks).

Reply:
Using the `LV_DRAW_BUF_*_STATIC` macros would be really better as they are more professional and  showcases a good practice.

Reply:
@FASTSHIFT please don't forget this one.

Reply:
@kisvegabor Done